components/contactPicker/index.less

```diff
@@ -0,0 +1,76 @@
+.contact-picker {
+    background: #F6F7F7;
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    z-index: 1000;
+    overflow: hidden;
+    display: none;
+
+    &.active {
+        display: block;
+    }
+
+    .inner {
+        position: fixed;
+        top: @header-height;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        overflow-y: auto;
+        z-index: 1;
+    }
+}
+
+.contact-list {
+    color: #6d8c9b;
+    font-weight: 500;
+
+    .letter-header {
+        font-size: 13px;
+        padding: 0.1em 10px;
```

Comment:
That seems way too high. This is 13px:

![image](https://cloud.githubusercontent.com/assets/206379/5672694/c1f5d986-9747-11e4-9b5d-f140636ba570.png)